packages/devtools/client/components/ModuleInstallList.vue

```diff
@@ -71,8 +79,11 @@ const openInEditor = useOpenInEditor()
           role="button"
           :info="item"
           mb2 h-full class="hover:bg-active!"
+          :class="{
+            'text-green border-green': item.installed,
+          }"
```

Comment:
It's a nice idea to have it able to list existing modules, one thing I wanted to do but leaked in the initial implementation.

I think only changing it to green feels a bit too implicit. I would think:

- Instead of the color, we should have a badge says "installed". We can use a slot to do that.
- We should find a way to differentiate transitive modules. For example, `vueuse` is installed but not listed in the config before it's contributed by devtools-ui package. For that cases, we are not able to "remove" it, so we should disable the uninstall action and state the reason.
- While the title of the page says "Install Module", clicking the module would lead to either "install" or "uninstall" action could lead to confusion. I would say we could disable the install action for the existing modules, and add a button (or hover state of the "installed" badge) for the uninstall action. Similar to VS Code extensions:

<img width="536" alt="Screenshot 2023-05-11 at 06 43 42" src="https://github.com/nuxt/devtools/assets/11247099/0099c82c-cfa9-4161-b31a-a1d5ad87d350">

Reply:
Yes, please feel free to push it directly. For modules in layers, am thinking the same is that we just disable uninstall.